cypress/e2e/MyOwnTests/forgot-password.spec.js

```diff
@@ -0,0 +1,28 @@
+/// <reference types = "cypress" />
+import {ForgotPassword} from '../../../pages/Forgot-password.page'
+import {RegisterPage} from '../../../pages/Register.page'
+import {LoginPage} from '../../../pages/Login.page'
+describe('Forgot password page tests with PageObject', ()=> {
+const forgotPassword = new ForgotPassword()
+const registerPage = new RegisterPage ()
+const loginPage = new LoginPage()
+
```

Comment:
```suggestion
beforeEach(()=> forgotPassword.forgotPasswordPageOpen() )
```
в кожному тесті перша дія однакова, тому можна це зробити в beforeEach hook